pkg/controller/worker/machine_images.go

```diff
@@ -46,28 +46,29 @@ func (w *workerDelegate) UpdateMachineImagesStatus(ctx context.Context) error {
 	return nil
 }
 
-func (w *workerDelegate) findMachineImage(name, version string) (urn, id, communityGalleryImageID *string, acceleratedNetworking *bool, err error) {
+// return sharedGalleryImageID as well
```

Comment:
remove the comment

Reply:
sorry, didn't notice this one. It is now removed